gui/default/syncthing/core/syncthingController.js

```diff
@@ -240,6 +242,46 @@ angular.module('syncthing.core')
             }
         });
 
+        $scope.$on(Events.DEVICE_REJECTED, function (event, arg) {
+            var pendingDevice = {
+                deviceID: arg.data.device,
+                time: arg.time,
+                name: arg.data.name,
+                address: arg.data.address
+            };
+            console.log("rejected device:", pendingDevice);
+
+            const i = $scope.pendingDevices.findIndex(function (d) {
+                return d.deviceID === pendingDevice.deviceID;
+            });
+            if (i > -1) $scope.pendingDevices[i] = pendingDevice;
```

Comment:
We usually don't do inline-ifs, even for one-liners.

Reply:
Fixed in a820c736851b393bf77460851fe17ce75d7e52de.